yt_dlp/extractor/youtube.py

```diff
@@ -3547,7 +3555,7 @@ class YoutubeTabIE(YoutubeBaseInfoExtractor):
         'url': 'https://www.youtube.com/feed/watch_later',
         'only_matching': True,
     }, {
-        'note': 'Recommended - redirects to home page',
+        'note': 'Recommended - redirects to home page.',
```

Comment:
😄

Reply:
lol ignore that (was reverting a change I had made there and must've missed this)